l10n_it_account_stamp/models/account_move.py

```diff
@@ -21,7 +21,7 @@ def is_tax_stamp_applicable(self):
             raise UserError(_("Missing tax stamp product in company settings!"))
         total_tax_base = sum(
             (
-                inv_tax.price_subtotal
+                abs(inv_tax.balance)
```

Comment:
Si suppone quindi che la valuta dell'azienda sia EUR, ma se non fosse così?
Credo sarebbe più corretto controllare prima che la valuta dell'azienda sia EUR, oppure controllare il valore convertito in EUR.

Reply:
`balance` è nella valuta della company che non per forza deve essere EUR
`stamp_apply_min_total_base` questo campo in prodotto è un float che dovresti scrivere in base alla tua company, forse è proprio questo campo che ti può far supporre che l'azienda sia EUR